src/components/CompareResults/ResultsTable.tsx

```diff
@@ -45,6 +45,16 @@ const cellsConfiguration: CompareResultsTableConfig[] = [
 
     gridWidth: '1fr',
   },
+  {
+    name: 'Standard Deviation Base',
+    key: 'standevbase',
+    gridWidth: '1fr',
+  },
+  {
+    name: 'Standard Deviation New',
+    key: 'standevnew',
+    gridWidth: '1fr',
+  },
```

Comment:
Good job! 

However, could you please set the standdevbase and standdevnew column header to `text-align: center` so it looks better?